fe/fe-core/src/main/java/com/starrocks/backup/mv/MVRestoreUpdater.java

```diff
@@ -205,4 +206,70 @@ public static void restoreBaseTableVersionMap(
         }
         baseTableVisibleVersionMap.put(localBaseTable.getId(), versionMap);
     }
+
+    public static boolean restoreBaseTableInfoIfNoRestored(Database restoreDb,
```

Comment:
suggest to rename this function to shouldRestoreBaseTableInfoIfNeeds?

Reply:
`restoreBaseTableInfoIfNoRestored ` means to restore base table when the base table has not been backed up and restored.

@reture success or not:
- true means restore sucess if the base table existed in the db, 
- false means restore failed because the base table dosn't existed.